libvast/vast/fbs/coder.fbs

```diff
@@ -0,0 +1,40 @@
+include "bitmap.fbs";
+
+namespace vast.fbs.coder.detail;
+
+table Base {
```

Comment:
Why is base a detail? It's simply a sequence of values. IMO it should be struct, as it will never change (by design).

Reply:
It's a detail in that it is an implementation detail of `coder` for now.

It also must be a table rather than a struct since it contains a vector of unknown size.